<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>服务端消息：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>691</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>服务器操作</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuQuery">
    <property name="title">
     <string>机票查询</string>
    </property>
    <addaction name="actionConditionQuery"/>
    <addaction name="actionAllQuery"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionShow"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOpetion">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionDelete"/>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuQuery"/>
   <addaction name="menuOpetion"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionExit"/>
   <addaction name="actionConditionQuery"/>
   <addaction name="actionAllQuery"/>
   <addaction name="actionAdd"/>
   <addaction name="actionDelete"/>
   <addaction name="actionUpdate"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/connect.png</normaloff>:/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>开启服务器</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/scissors.png</normaloff>:/images/scissors.png</iconset>
   </property>
   <property name="text">
    <string>关闭服务器</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/power-off.png</normaloff>:/images/power-off.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionConditionQuery">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/download.png</normaloff>:/images/download.png</iconset>
   </property>
   <property name="text">
    <string>查询特定航班</string>
   </property>
  </action>
  <action name="actionAllQuery">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/download.png</normaloff>:/images/download.png</iconset>
   </property>
   <property name="text">
    <string>查询所有航班</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>显示内容</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
   </property>
   <property name="text">
    <string>增加航班信息</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除航班信息</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/images/update.png</normaloff>:/images/update.png</iconset>
   </property>
   <property name="text">
    <string>更新航班信息</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
